riotfile.py

```diff
@@ -1153,6 +1153,45 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             },
             command="pytest {cmdargs} tests/contrib/aiopg",
         ),
+        Venv(
+            name="aiohttp",
+            command="pytest {cmdargs} tests/contrib/aiohttp",
+            pkgs={"pytest": [">=3"], "pytest-aiohttp": [latest]},
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version="3.5", max_version="3.6"),
+                    pkgs={
+                        "aiohttp": ["~=2.0", "~=2.1", "~=2.2"],
+                        "aiohttp_jinja2": ["~=0.12", "~=0.13"],
+                        "yarl": "~=0.18.0",
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.5", max_version="3.8"),
+                    pkgs={
+                        "aiohttp": "~=2.3",
+                        "aiohttp_jinja2": "~=0.15",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.5", max_version="3.7"),
+                    pkgs={
+                        "aiohttp": ["~=3.0.0", "~=3.1.0", "~=3.2.0", "~=3.3.0", "~=3.5.0", "~=3.6.0"],
+                        "aiohttp_jinja2": "~=0.15.0",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.8"),
+                    pkgs={
+                        "aiohttp": ["~=3.0.0", "~=3.1.0", "~=3.2.0", "~=3.3.0", "~=3.6.0"],
+                        "aiohttp_jinja2": "~=0.15.0",
+                        "yarl": "~=1.0",
+                    },
+                ),
```

Comment:
these look almost the same, aiohttp >=3.0.0, but they have different Python constraints

they are both missing `3.4.0`, and the second is missing `3.5.0` ?